src/index.js

```diff
@@ -25,4 +26,23 @@ const register = (trackDef, { force = false } = {}) => {
   }
 };
 
+const register = (trackDef, ...args) => {
+  // eslint-disable-next-line no-console
+  console.warn('register() is deprecated. Please use registerTrack() instead.');
```

Comment:
Please, let's not do this. HiGlass is logging enough meaningless warnings already. To avoid those we'd have to update **ALL** existing plugin tracks. I do not want to do this. Instead I propose:

```javascript
const register = (definition, { track: true, dataFetcher: false, ...options } = {}) => {
  if (dataFetcher) registerDataFetcher(definition, { ...options });
  else registerTrack(definition, { ...options });
};
```

Reply:
I have added a `pluginType` option, which may make it easier to add more plugin types in the future in https://github.com/higlass/higlass-register/pull/2/commits/ab97854ea964b3f3b849cc39612ed2e083b081ec Would you prefer the boolean method instead?

Reply:
Good catch. `pluginType` makes more sense.